zou/app/services/tasks_service.py

```diff
@@ -192,29 +192,38 @@ def get_tasks_for_asset(asset_id):
     return get_task_dicts_for_entity(asset.id)
 
 
+def get_tasks_for_episode(episode_id):
+    """
+    Get all tasks for given episode.
+    """
+    episode = shots_service.get_episode_raw(episode_id)
+    return get_task_dicts_for_entity(episode.id)
+
+
 def get_task_dicts_for_entity(entity_id):
     """
     Return all tasks related to given entity. Add extra information like
     project name, task type name, etc.
     """
-    query = Task.query.order_by(Task.name) \
```

Comment:
If you can keep the previous code style, it's better. It would be more coherent with the rest.

Reply:
Done